server/src/test/java/org/elasticsearch/search/aggregations/bucket/histogram/NumericHistogramAggregatorTests.java

```diff
@@ -100,6 +106,41 @@ public void testDoubles() throws Exception {
         }
     }
 
+    public void testDates() throws Exception {
```

Comment:
I'm not sure we should support this, but we do currently, so I wanted a test for the legacy case.